internal/web/api/api.go

```diff
@@ -166,7 +172,109 @@ func getClusteringPeersHandler(host service.Host) http.HandlerFunc {
 	}
 }
 
-func liveDebugging(_ service.Host, callbackManager livedebugging.CallbackManager) http.HandlerFunc {
+type dataKey struct {
+	ComponentID livedebugging.ComponentID
+	Type        livedebugging.DataType
+}
+
+func graph(_ service.Host, callbackManager livedebugging.CallbackManager, logger log.Logger) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var moduleID livedebugging.ModuleID
+		if vars := mux.Vars(r); vars != nil {
+			moduleID = livedebugging.ModuleID(vars["moduleID"])
+		}
+
+		window := setWindow(w, r.URL.Query().Get("window")) // in seconds
```

Comment:
```suggestion
		windowSeconds := setWindow(w, r.URL.Query().Get("window"))
```
Generally if we can make code more readable without comments, it's better to do that - comments may be treated as a duct tape to help make sense of a code that is otherwise not too readable ;)

Reply:
Actually, can't `window` become a `time.Duration` right away please?

Reply:
we use it as a float to calculate the rate:
data.Rate = float64(data.Count) / float64(window)

Reply:
I don't think it's a problem that cannot be overcome if window is a duration